### PR TITLE
Modules v2.1 enhancements

### PR DESCRIPTION
**Pull request by mathanmahe**
…in parameters

### CODE REVIEW

**Review by wli-linda — COMMENTED** [review]
Nice addition of constraints, LGTM!

File: src/main/java/seedu/allonus/modules/StudyManager.java [hunk 1]
```diff
@@ -399,7 +432,9 @@ public Module addModuleParser(String userInput) {
         //add m/CS2113 c/lec d/Thursday t/2pm-4pm
         try {
             String[] rawInput = userInput.split(" ", 2);
-            String[] parameters = rawInput[1].split(" ", 4);
+            //String[] parameters = rawInput[1].split(" ", 4);
+            //String[] parameters = rawInput[1].split("[mcdt]/ ", 4);
```
Comment (YitHien) [inline src/main/java/seedu/allonus/modules/StudyManager.java:436]: Do consider removing unused code comments!

**Review by YitHien — COMMENTED** [review]
Nice additions of checks and constraints! LGTM

**Review by OzairHasan — COMMENTED** [review]
LGTM

**Pull request merged**